src/Sfx/App/Scripts/Services/RestClientService.ts

```diff
@@ -571,6 +571,17 @@ module Sfx {
             return this.getEvents(FabricEvent, url, null, null, messageHandler);
         }
 
+        public restartReplica(nodeName: string, partitionId: string, replicaId: string, messageHandler?: IResponseMessageHandler): angular.IPromise<{}> {
+            let url = "Nodes/"
+                + nodeName + "/$/"
+                + "GetPartitions/"
+                + partitionId + "/$/"
+                + "GetReplicas/"
+                + replicaId + "/$/"
+                + "Restart";
+            return this.post(this.getApiUrl(url), "Restart replica", null, messageHandler);

```

Comment:
This API requires api-version=6.0
By default the api-version is 3.0 so I am not sure if this works.

Reply:
I set it to 6.0